src/api/app/models/github_payload/push.rb

```diff
@@ -2,6 +2,10 @@
 class GithubPayload::Push < GithubPayload
   def payload
     payload_ref = webhook_payload.fetch(:ref, '')
+
+    # If ref is deleted we use the "before" commit sha. The "after" commit sha is 0* in this case.
+    yaml_reference = webhook_payload[:deleted] ? webhook_payload[:before] : webhook_payload[:after]
```

Comment:
I'm just not sure if you can do the check this way. Is this converted to a boolean value, or is it a string with true or false?

Reply:
That's in my TODO, yes.

Reply:
alright :+1:

Reply:
I tested with a real payload and we receive a Boolean, not a String. We can keep this code.

Reply:
ah nice, so its smart enough to do so